articles/libraries/lock/v10/index.md

```diff
@@ -5,7 +5,6 @@ title: Lock 10 for Web
 description: A widget that provides a frictionless login and signup experience for your web apps.
 img: media/articles/libraries/lock-web.png
 ---
-
 # Lock 10 for Web
 
 You're looking at the documentation for the _easiest_ way of securing your website and mobile apps!
```

Comment:
Can you rewrite this intro? Looks like a marketing doc and it shouldn't, this is reference documentation.

I would change it to sth like:

```
Lock is an embeddable login form, [configurable to your needs][lock-customization] and ready for use on web apps. It enables you to easily add social identity providers to Lock, allowing your users to login seamlessly using any provider they want.
```